recipes/sos-bioinfo/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sos-bioinfo" %}
+{% set version = "0.9.10.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "47fdd8399512a4a7f98e97225f4811c6aa73787cad8713c513a3e9cc65417f63" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [not py36]
```

Comment:
Can this package be a `noarch: python` package? And the others as well?
If so you can skip python 36 and probably 37? as well with 

```
  host:
    - python <3.5
```

Reply:
sos only works for python 3.6 and higher.